aea/cli/check_packages.py

```diff
@@ -595,24 +600,20 @@ def check_pypi_dependencies(configuration_file: Path) -> None:
 
 
 @click.command(name="check-packages")
-@click.argument(
-    "packages_dir",
-    type=click.Path(dir_okay=True, exists=True),
-    default=Path.cwd() / "packages",
-)
```

Comment:
No point in using this flag, we should utilise `--registry-path` flag from the top level command group